connectors/bindings/lbc.go

```diff
@@ -49,16 +49,16 @@ type LiquidityBridgeContractPegOutQuote struct {
 
 // LiquidityBridgeContractProvider is an auto generated low-level Go binding around an user-defined struct.
 type LiquidityBridgeContractProvider struct {
-	Id                      *big.Int
-	Provider                common.Address
-	Name                    string
-	Fee                     *big.Int
-	QuoteExpiration         *big.Int
-	AcceptedQuoteExpiration *big.Int
-	MinTransactionValue     *big.Int
-	MaxTransactionValue     *big.Int
-	ApiBaseUrl              string
-	Status                  bool
+	Id                      *big.Int `json:"id" example:"1" description:"Provider id"`
```

Comment:
I think this annotations will be lost when code is re generated

Reply:
The lbc.go already contains fair amount of differences from ones being generated, best practise is to copy paste functions to old lbc.go and replace the ABI to not break server from different variable names being generated.